src/index.html

```diff
@@ -4,10 +4,42 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&amp;display=swap" rel="stylesheet">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+      <div class="card" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="card__image"
+        >
+        <div class="card__information">
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__code">Product code: 195434</p>
```

Comment:
row is too long
`<h2 class="card__title">`
`APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)`
`</h2>`